<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>544</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>706</width>
    <height>544</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>706</width>
    <height>544</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MoreCoasts       Nuevo Cálculo</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>FormatFactoryAgregar un título.ico</normaloff>FormatFactoryAgregar un título.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(201, 230, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame_NuevoCalculo">
      <property name="styleSheet">
       <string notr="true">QFrame{
background-color: rgb(201, 230, 255);
}
QLabel{
background-color: #000000ff;
color::rgb(0, 74, 116);
border:8px solid#000000ff;
}
QLineButton{
border:0px;
color:rgb(255, 255, 255);
border-bottom:2px solid rgb(61,61,61);
}
QPushButton{
background-color:rgb(0, 74, 116);
border-radius:15px;
color: rgb(255, 255, 255);
}

QLineEdit{
background-color: rgb(255, 255, 255);
}
QPushButton:hover{
background-color: rgb(0, 12, 255);
border-radius:15px;
color: rgb(0, 0, 0);
}

</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_NuevoCalculo">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>671</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial Black</family>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 74, 116);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Nuevo Cálculo&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>110</y>
         <width>681</width>
         <height>43</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <family>Verdana</family>
            <pointsize>8</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(143, 170, 163);</string>
          </property>
          <property name="text">
           <string>Los campos con (*)  son obligatorios</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>400</y>
         <width>671</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_Error">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 1, 5);</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Label Error&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>460</y>
         <width>671</width>
         <height>43</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="but_GuardarNuevoCalculo">
          <property name="minimumSize">
           <size>
            <width>91</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>91</width>
            <height>31</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Verdana</family>
            <pointsize>9</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
background-color: rgb(71, 74, 255);
color: rgb(0, 0, 0);
border-radius:8px;
}

QPushButton:hover{
background-color: rgb(16, 132, 255);
border-top-left-radius:12px;
border-bottom-left-radius:12px;

}</string>
          </property>
          <property name="text">
           <string>Guardar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="butCancelarNuevoCalculo">
          <property name="minimumSize">
           <size>
            <width>91</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>91</width>
            <height>31</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Verdana</family>
            <pointsize>9</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
color: rgb(255, 255, 255);
	background-color: rgb(221, 1, 5);
border-radius:8px;
}

QPushButton:hover{
background-color: rgb(255, 120, 129);
border-top-left-radius:12px;
border-bottom-left-radius:12px;

}</string>
          </property>
          <property name="text">
           <string>Cancelar</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>681</width>
         <height>53</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>51</width>
            <height>51</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>51</width>
            <height>51</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>Agregar_un_título-removebg-preview.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_SGCTS_GeoCuba">
          <property name="minimumSize">
           <size>
            <width>180</width>
            <height>51</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>180</width>
            <height>51</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Cooper Black</family>
            <pointsize>16</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #000000ff;
color: rgb(0, 0, 0);</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;MoreCoasts&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="label_ALtura">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>310</y>
         <width>223</width>
         <height>48</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Altura (Hb)*</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_DensidadMar">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>203</y>
         <width>223</width>
         <height>48</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Densidad del Mar (ρs)*</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_CoeficientePorocidad">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>257</y>
         <width>223</width>
         <height>47</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Coeficiente de Porocidad (n)*</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_DensidadArena">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>149</y>
         <width>223</width>
         <height>48</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Densidad de Arena (ρ)*</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_altura">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>310</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>121</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>121</width>
         <height>41</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="maxLength">
        <number>50</number>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_densidadArena">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>160</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>121</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>121</width>
         <height>41</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>IBeamCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="maxLength">
        <number>50</number>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_CoeficientePorocidad">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>260</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>121</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>121</width>
         <height>41</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="maxLength">
        <number>50</number>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_DensidadMar">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>210</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>121</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>121</width>
         <height>41</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="maxLength">
        <number>50</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_Ubicacion_2">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>310</y>
         <width>198</width>
         <height>48</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Ubicación*</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_AnguloR">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>149</y>
         <width>198</width>
         <height>48</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Ángulo de Rompiente (α)*</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_IndiceR">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>203</y>
         <width>198</width>
         <height>48</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Índice de Rompiente(k)*</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_Ubicacion">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>257</y>
         <width>198</width>
         <height>47</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Municipio*</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBoxUbicCalculo">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>340</y>
         <width>130</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>121</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>130</width>
         <height>22</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <item>
        <property name="text">
         <string>Seleccione una Ubicación</string>
        </property>
       </item>
      </widget>
      <widget class="QLineEdit" name="lineEdit_IndiceRompiente">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>210</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>121</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>121</width>
         <height>41</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="maxLength">
        <number>50</number>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_AnguloRompiente">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>157</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>121</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>121</width>
         <height>41</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="maxLength">
        <number>50</number>
       </property>
      </widget>
      <widget class="QPushButton" name="but_SeleccionarMunCalculo">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>297</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>91</width>
         <height>31</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>91</width>
         <height>31</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
color: rgb(0, 0, 0);
background-color: rgb(0, 170, 255);
border-radius:8px;
}

QPushButton:hover{
background-color: rgb(0, 196, 255);
border-top-left-radius:12px;
border-bottom-left-radius:12px;

}</string>
       </property>
       <property name="text">
        <string>Seleccionar</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>263</y>
         <width>130</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>121</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>130</width>
         <height>22</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <item>
        <property name="text">
         <string>Seleccione un Municipio</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_DensidadArena_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>350</y>
         <width>223</width>
         <height>48</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Medición Práctica (P)*</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_PNuevo">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>360</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>121</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>121</width>
         <height>41</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="maxLength">
        <number>50</number>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
